modules/nw-creating-anp-egress-policy.adoc

```diff
@@ -0,0 +1,203 @@
+//Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/ovn-k-nodes-egress-nodes-networks-peer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-creating-anp-egress_{context}"]
+= Creating an AdminNetworkPolicy to monitor egress
+
+You configure an egress policy by creating an `AdminNetworkPolicy` custom resource (CR) object. The egress policy should match network traffic that meets any of the following criteria:
+
+* An IP address range in CIDR format
+
+* A DNS name that resolves to an IP address
+
+* A port number
+
+* A protocol that is one of the following protocols: TCP, UDP, and SCTP
+
+[NOTE]
+====
+`Nodes` and `networks` peer only work for egress traffic.
+====
+
+.Prerequisites
+
+* You are logged in to the cluster with a user with administrator privileges for the project.
+* You installed the `oc` CLI.
+
+.Procedure
+
+. Create a YAML file to define the egress `AdminNetworkPolicy` CR using nodes peer:
++
+[source,terminal]
+----
+oc create -f <policy_name>.yaml
```

Comment:
I think you mean creation of yaml for the policy, so it should be use any editor of choice.
vi <policy_name>.yaml
You may also remove thisline.